app/scripts/modules/netflix/fastProperties/domain/scope.domain.spec.ts

```diff
@@ -0,0 +1,30 @@
+import {IPlatformProperty} from './platformProperty.model';
+import {Scope} from './scope.domain';
+
+
+describe('Scope Domain Spec', function () {
+
+  describe('statically create a new scope from a scope fetched from the props API', () => {
+    it('should build a new scope when a we have complete scope from props API ', function () {
+      let platformProperty = <IPlatformProperty>{
```

Comment:
If you did `let platformProperty: IPlatformProperty` would it still require the cast?